dragonchain/webserver/routes/smart_contracts.py

```diff
@@ -64,6 +65,13 @@ def list_contract_v1() -> Tuple[str, int, Dict[str, str]]:
     return helpers.flask_http_response(200, smart_contracts.list_contracts_v1())
 
 
+@request_authorizer.Authenticated()
+def get_smart_contract_logs_v1(contract_id: str) -> Tuple[str, int, Dict[str, str]]:
+    since = flask.request.args.get("since")
+    tail = flask.request.args.get("tail")
```

Comment:
This will always be a string. All request args are always strings. You'll want to parse this into an int and throw a reasonable error (400) if it's not parsable into an int.

I'm surprised mypy didn't pick this up. I guess flask isn't typed or something.

Reply:
Demo:
```python
import flask
app = flask.Flask(__name__)

@app.route("/")
def test():
    return str(type(flask.request.args.get("test")))

app.run()
```

```sh
$ curl localhost:5000/?test=123
<class 'str'>
```